src/header/header.scss

```diff
@@ -25,159 +25,105 @@
-          font-size: $iui-font-size;
-        }
-
-        .iui-header-button-icon {
-          margin: 0 $iui-l + $iui-xs;
-        }
-      }
-
-      .iui-header-button-icon {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin: 0 $iui-xs;
-        display: inline-flex;
-      }
-    }
-
-    > .iui-right {
-      .iui-user-icon {
-        @include iui-user-icon-size($size: $iui-icons-large, $font-size: $iui-font-size-small);
-      }
-    }
-  }
+@mixin iui-page-header-center {
+  @include iui-page-header-zones;
+}
 
-  @media (max-width: 768px) {
-    &,
-    &.iui-slim {
-      .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            @include iui-display-none;
-          }
-
-          .iui-header-button-icon + .iui-button-label {
-            margin-left: 0;
-          }
-        }
-      }
-    }
-  }
+@mixin iui-page-header-right {
+  @include iui-page-header-zones;
 
-  @media (max-width: 1024px) {
-    .iui-left {
-      .iui-header-logo > .iui-label {
-        display: none;
+  @media (prefers-reduced-motion: no-preference) {
+    .iui-user-icon {
+      &,
+      * {
+        transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, font-size $iui-speed-fast ease-out;
       }
     }
   }
 }
+// #endregion Selection
 
-@mixin iui-header-button-icon($size: $iui-icons-large, $border-radius: $iui-border-radius) {
-  width: $size;
-  height: $size;
-  border-radius: $border-radius;
-  @media (prefers-reduced-motion: no-preference) {
-    transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, margin $iui-speed-fast ease-out;
-  }
+// #region Header buttons
+@mixin iui-header-button-reset {
+  height: 100%;
+  border-radius: 0 !important;
 }
 
-@mixin iui-header-buttons {
-  .iui-header-button-icon {
-    @include iui-header-button-icon;
-  }
+@mixin iui-header-brand {
+  @include iui-header-button-reset;
+  display: inline-flex;
+  align-items: center;
+  flex-shrink: 0;
+  white-space: nowrap;
+  background-color: transparent;
 
-  // boost specificity
-  .iui-header-button.iui-header-button {
-    @include iui-header-button;
+  &-icon {
+    width: $iui-icons-large;
+    height: $iui-icons-large;
+    margin: 0 $iui-l;
+    transition: fill $iui-speed-fast ease-out;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out, margin $iui-speed-fast ease-out,
+        fill $iui-speed-fast ease-out;
+    }
+    @include themed {
+      fill: t(iui-icons-color-actionable);
+    }
   }
 
-  .iui-header-split-button {
-    @include iui-header-split-button;
-  }
-}
+  &-label {
+    margin-right: $iui-sm;
+    font-size: $iui-font-size-leading;
+    transition: color $iui-speed-fast ease-out;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: color $iui-speed-fast ease-out, font-size $iui-speed-fast ease-out;
```

Comment:
when color is changed?

Reply:
Really seems to only be when switching themes.  Removing color transition here and fill transition for the brand icon for the same reason (although it is visually inconsistent with the other buttons in the header).